styles.css

```diff
@@ -383,3 +383,7 @@ td.tight {
 .mouseover-reveal:hover {
   opacity: 1;
 }
+
+table a {
```

Comment:
Do we need this blanket rule when `.text-decoration-none` exists?

Reply:
That doesn't work in tables with markdown generated links (specifically the requirements list in this PR). I suppose we could add the class to the generated link tag in the renderer but it would essentially accomplish the same thing.